test/test_jit.py

```diff
@@ -5230,6 +5275,21 @@ def test_pow_int(x, y):
         self.checkScript(test_pow_float, (2.0, 2.0))
         self.checkScript(test_pow_int, (2.0, 2))
 
+        num_list = [-1000000, -1000, -2, 0, 1, 3, 1000, 10000000]
```

Comment:
Range of inputs looks good - could you test all of the functions with a range instead of just isnan?